exasol_data_science_utils_python/schema/table_like_name.py

```diff
@@ -0,0 +1,8 @@
+from abc import ABC
+
+from exasol_data_science_utils_python.schema.dbobject_name_with_schema import DBObjectNameWithSchema
+
+
+class TableLikeName(DBObjectNameWithSchema, ABC):
+    """Abstract DBObjectName for table like objects, such as Tables and Views"""
+    pass
```

Comment:
if you have a docstring, you don't need `pass` anymore

Reply:
done